### ISSUE TITLE
Run from pre-built docker image

### ISSUE THREAD

**Issue by luiztauffer** [qdrant/qdrant-web-ui#115]
A Qdrant admin UI is an amazing idea! It would be perfect if we could just run it from a pre-built docker image though...
Even better would be to have a version of qdrant vector store docker image with the UI served there as well, in a separate port.

**Comment by timvisee** [comment]
I think what you're suggesting is already available.

If you run the Qdrant Docker container with

```bash
docker run -p 6333:6333 qdrant/qdrant
```

the dashboard is available at: http://localhost:6333/dashboard

This URL is also printed in the log when the container starts.

Or are you asking something else?

**Comment by luiztauffer** [comment]
@timvisee you're right, I missed the fact that there is already a dashboard! thanks!
